corehq/apps/es/es_query.py

```diff
@@ -167,9 +167,9 @@ def __init__(self, index=None, debug_host=None, es_instance_alias=ES_DEFAULT_INS
         self._aggregations = []
         self.es_instance_alias = es_instance_alias
         self.es_query = {"query": {
-            "filtered": {
-                "filter": {"and": []},
-                "query": queries.match_all()
+            "bool": {
+                "filter": queries.match_all(),
+                "must": []
```

Comment:
This change also breaks `_query`, `set_query`, and `get_query` in this class, which expect the old format.

Reply:
@esoergel I know that these changes break a lot of in application, I prepared this PR to make sure that we go to correct direction which Dimagi want to go.